BikeRentalApi/urls.py

```diff
@@ -6,19 +6,20 @@
 
 
 urlpatterns = [
-    path('bikes/', bikes_list, name = 'bikes_list'),
-    path('bikes/<int:pk>/', bikes_detail, name = 'bikes_detail'),
+    path('bikes', bikes_list, name = 'bikes_list'),
+    path('bikes/<int:pk>', bikes_detail, name = 'bikes_detail'),
     path('bikes/rented', bikes_rented, name = 'bikes_rented'),
     path('bikes/blocked', bikes_blocked, name = 'bikes_blocked'),
     path('bikes/blocked/<int:pk>', bikes_unblocked, name = 'bikes_unblocked'),
-    path('stations/', stations_list, name = 'station_list'),
-    path('stations/<int:pk>/', stations_detail, name = 'station_detail'),
-    path('stations/<int:pk>/bikes/', stations_detail_bikes, name = 'station_detail_bikes'),
+    path('stations', stations_list, name = 'station_list'),
+    path('stations/<int:pk>', stations_detail, name = 'station_detail'),
+    path('stations/<int:pk>/bikes', stations_detail_bikes, name = 'station_detail_bikes'),
     path('stations/blocked', stations_blocked, name = 'bikes_blocked'),
+    path('stations/active', stations_blocked, name = 'bikes_blocked'),
     path('stations/blocked/<int:pk>', stations_blocked_detail, name = 'bikes_blocked_detail'),
-    path('register/', auth_views.register, name = 'register'),
-    path('login/', auth_views.login, name = 'login'),
-    path('logout/', auth_views.logout, name = 'logout'),
-    path('techs/', techs_list, name = 'techs_list'),
+    path('register', auth_views.register, name = 'register'),
+    path('login', auth_views.login, name = 'login'),
+    path('logout', auth_views.logout, name = 'logout'),
+    path('techs', techs_list, name = 'techs_list'),
     path('techs/<int:pk>', techs_detail, name = 'techs_detail'),
```

Comment:
This doesn't work with our current frontend, which always appends the trailing slash (and I don't know how to change it)

Reply:
But that not backend problem

Reply:
For it to pass postman tests, trailing slashes must be removed

Reply:
Both backend and frontend need to pass the tests. If the solution to this is not obvious, I think we ought to discuss this more thoroughly in private

Reply:
Never mind, its working in both cases now